.github/ISSUE_TEMPLATE/code_improvement.md

```diff
@@ -2,7 +2,7 @@
 name: Code improvement
 about: Suggest a code improvement, i.e. refactoring, deprecation, etc.
 title: ''
-labels: enhancement, help wanted, refactors / code health
+labels: enhancement, refactors / code health
```

Comment:
why does this one have `enhancement` too?

Reply:
I think the idea is code enhancement, but we could also remove